pages/linux/llvd.md

```diff
@@ -0,0 +1,20 @@
+# llvd
+
+> Linkedin Learning Video Downloader.
+> More information: <https://github.com/knowbee/llvd>.
+
+- Download a Course:
+
+`llvd -c {{course-slug}} --cookies`
+
+- Download a Path with throttling:
```

Comment:
```suggestion
- Download a course [p]ath with [t]hrottling between 10 to 30 seconds:
```
I think mentioning seconds is essential in this command description as it gives context to the throttling flag.